Cargo.toml

```diff
@@ -33,7 +33,7 @@ serde = { version = "1.0.101", optional = true }
 byteorder = { version = "0.5", optional = true }
 
 [features]
-"ardupilotmega" = []
+"ardupilotmega" = ["common", "icarous", "uavionix"]
 "asluav" = []
```

Comment:
If we add default features for `ardupilotmega`, wouldn't it make sense to add default features for all of the others as well?

Reply:
Done.